test/views/TwoEyeReviewResultViewSpec.scala

```diff
@@ -227,6 +239,17 @@ class TwoEyeReviewResultViewSpec extends ViewSpecBase {
       errorMessageSpan.text.replaceAll("\u00a0", " ") shouldBe s"Error: $errorMsg".replaceAll("&nbsp;", " ")
     }
 
+    "Add id of error message span to the aria field of fieldset" in new Test {
+
+      val boundForm: Form[ApprovalStatus] = form.bind(emptyFormData)
```

Comment:
This test seems to be identical to the earlier one. I think we should be submitting an invalid value rather than an empty form.

Reply:
Its two differing error situations, the test confirms that the aria-describedby is set correctly both situations. The empty form is the only error which can occur within our usage